src/main/java/org/broadinstitute/dsm/export/ExportToES.java

```diff
@@ -124,11 +130,20 @@ private static void exportTissueRecords(int instanceId) {
 
     public static void checkWorkflowNamesAndExport(List<String> workFlowColumnNames, List<ParticipantDataDto> allParticipantData, int instanceId) {
         DDPInstance ddpInstance = DDPInstance.getDDPInstanceById(instanceId);
+        Optional<String> maybeEsParticipantIndex =
+                new DDPInstanceDao().getEsParticipantIndexByInstanceId(instanceId);
         for (ParticipantDataDto participantData: allParticipantData) {
             String data = participantData.getData();
             if (data == null) {
                 continue;
             }
+            String ddpParticipantId = participantData.getDdpParticipantId();
+            if (ddpParticipantId.length() > 20) {
```

Comment:
there is a method already in `ParticipantUtil` `isGuid` which returns boolean whether id is GUID or not, which will bring more clarification what you are doing here.